css/style.css

```diff
@@ -478,3 +493,207 @@ textarea {
+}
+
+/* DESKTOP VERSION */
+
+@media (min-width: 760px) {
+  /* HEADER SECTION */
+
+  .menu {
+    display: flex;
+    list-style: none;
+    gap: 32px;
+    margin-right: 10%;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .menu-link {
+    text-decoration: none;
+    color: #344563;
+  }
+
+  .toolbar {
+    align-items: center;
+  }
+
+  .toolbar i {
+    display: none;
+  }
+
+  .headline-container {
+    flex-direction: row;
+    margin-top: 0;
+    background-image: url("../assets/background-desk.png");
+    background-size: 100%;
+    background-position: 100% 10%;
+    align-items: none;
+    padding: 15% 16%;
+    border-bottom-left-radius: 50px;
+  }
+
+  .headline {
+    margin-top: 0;
+    padding: 0 10%;
+  }
+
+  .grettings {
+    margin-top: 0;
+  }
+
+  .desc {
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+    width: 70%;
+  }
+
+  /* WORKS SECTION */
+
+  .works-section {
+    grid-template-columns: minmax(300px, 70%);
+    justify-content: center;
+  }
+
+  .card-works {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+    gap: 25px;
+  }
+
+  .card-works:nth-child(even) .img-container {
+    order: 1;
+  }
+
+  .project-screenshot {
+    width: 100%;
+  }
+
+  .project-desc {
+    margin-top: 0;
+    width: 100%;
+  }
+
+  .project-desc h2 {
+    width: 100%;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #091e42;
+  }
+
+  .s-desc {
+    align-items: baseline;
+    justify-content: baseline;
+  }
+
+  .s-desc :first-child {
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .s-desc li {
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 24px;
+    color: #6b778c;
+  }
+
+  .desc-text {
+    margin-top: 10px;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .categories li {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+  }
+
+  .see-project-btn {
+    font-size: 17px;
+    line-height: 24px;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #396df2;
+  }
+
+  /* ABOUT SECTION */
+
+  .about-content {
+    flex-direction: row;
+    padding: 200px 100px;
+    gap: 25px;
+  }
+
+  .about-info {
+    flex: 1;
+    padding: 10px;
+  }
+
+  .technologies {
+    flex: 2;
+  }
+
+  .skills-container {
+    margin-top: 0;
+  }
+
+  .dropdown-list {
+    width: 50%;
+    flex-direction: row;
+    gap: 20px;
+  }
+
+  .dropdown-list-item {
+    flex-direction: column;
+    flex: 1;
+  }
+
+  .dropdown-list li:last-child span {
+    text-transform: capitalize;
+  }
+
+  /* CONTACT SECTION */
+
+  .contact-container {
+    align-items: center;
+    background-image: url("../assets/contact-bg-desk.png");
+    background-size: 100%;
+    background-position: 100% 10px;
+    padding-bottom: 300px;
+  }
+
```

Comment:
- [ ] Kindly readjust the `height` and the `width` of this form to stick to the Figma design :+1: 

Your design

![Screenshot from 2022-12-08 19-04-04](https://user-images.githubusercontent.com/80458199/206524750-bc3995bc-2d9a-443d-8d34-0f0d8628f64c.png)

Figma design

![Screenshot from 2022-12-08 19-07-55](https://user-images.githubusercontent.com/80458199/206524707-47e7f73e-da7d-4cb2-a291-8ee443c812e7.png)